src/items/functions.md

```diff
@@ -203,12 +203,30 @@ extern "C" fn new_i32() -> i32 { 0 }
 let fptr: extern "C" fn() -> i32 = new_i32;
 ```
 
-Functions with an ABI that differs from `"Rust"` do not support unwinding in the
-exact same way that Rust does. Therefore, unwinding past the end of functions
-with such ABIs causes the process to abort.
+### Unwinding
+
+Each non-`Rust` ABI string has a corresponding ABI with the suffix `-unwind`.
+
+The choice of ABI, together with the [panic mode][panic-modes], determines the
+behavior when unwinding out of a function.
+
+In the table below, "Unforced foreign unwind" refers to something like a C++
+exception; the table indicates the behavior when entering a Rust stackframe via
+an function or function pointer declared with the specified ABI string.
+Additionally, `C` in the ABI strings may be substituted with `stdcall` or any
+other ABI supported by the language implementation.
+
+| panic runtime  | ABI          | `panic`-unwind                        | Unforced foreign unwind |
+| -------------- | ------------ | ------------------------------------- | ----------------------- |
+| `panic=unwind` | `"C-unwind"` | unwind                                | unwind                  |
+| `panic=unwind` | `"C"`        | abort                                 | UB                      |
+| `panic=abort`  | `"C-unwind"` | `panic!` aborts                       | abort                   |
+| `panic=abort`  | `"C"`        | `panic!` aborts (no unwinding occurs) | UB                      |
```

Comment:
```suggestion
| `panic=abort`  | `"C-unwind"` | `panic!` aborts (no unwinding occurs) | abort                   |
| `panic=abort`  | `"C"`        | `panic!` aborts (no unwinding occurs) | UB                      |
```

Niko's suggestion also does this, but in case that gets dropped I would still like this part to be more consistent.